ecosystem/typescript/sdk/src/plugins/aptos_token.ts

```diff
@@ -0,0 +1,193 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+/* eslint-disable max-len */
+
+import { AptosAccount } from "../account/aptos_account";
+import { AnyNumber } from "../bcs";
+import { MAX_U64_BIG_INT } from "../bcs/consts";
+import { Provider } from "../providers";
+import { AptosClient, OptionalTransactionArgs } from "../providers/aptos_client";
+import { TransactionBuilderRemoteABI } from "../transaction_builder";
+import { getPropertyValueRaw } from "../utils/property_map_serde";
+
+/**
+ * Class for managing aptos_token
+ */
+export class AptosToken {
```

Comment:
Should we have some view functions here as well? And other functionalities such as transfer, burn, freeze/unfreeze, etc.